android/lib/src/main/java/com/airbnb/android/react/maps/AirMapManager.java

```diff
@@ -40,24 +39,10 @@
     );
 
     private ReactContext reactContext;
+    private final Context appContext;
 
-    private Context appContext;
-    private AirMapMarkerManager markerManager;
-    private AirMapPolylineManager polylineManager;
-    private AirMapPolygonManager polygonManager;
-    private AirMapCircleManager circleManager;
```

Comment:
👍  I put them there a long time ago, as there was some work that I was doing that required some cross-talk between them, but I don't think that code ever got committed.